lua/nvim-tree/actions/moves/item.lua

```diff
@@ -8,8 +8,9 @@ local M = {}
 
 ---@param where string
 ---@param what string
+---@param opts table|nil
 ---@return fun()
-function M.fn(where, what)
+function M.fn(where, what, opts)
```

Comment:
Moreover, if we all agree, I'd do a little refactor here, having just an `opts` table with the `where` and `what` fields, so that we can easily add new ones if necessary.

Reply:
Yes please. The API could be altered to pass `opts`.

We would need to retain backwards compatibility as per `ApiTreeToggleOpts` so as not to break existing users.

Reply:
Unless we release 0.x.x and introduce whatever breaking changes we want for 1.0.

Reply:
> Unless we release 0.x.x and introduce whatever breaking changes we want for 1.0.

Interesting... the highlight overhaul could also go before 1.0.0. Perhaps we might do an audit and do any "final breaks" before the big release.